Person-CRUD/Person.Data/PersonRepository.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Person_CRUD.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Data
+{
+    public class PersonRepository : IPersonRepository
+    {
+        private readonly CompanyContext _db;
+
+        public PersonRepository(CompanyContext db)
+        {
+            _db = db;
+        }
+
+        public async Task<List<Person>> GetAll()
+        {
+            return await _db.Persons.ToListAsync();
+        }
+        public async Task<bool> CreatePerson(Person person) {
+            _db.Persons.Add(person);
+            _db.SaveChanges();
```

Comment:
why you don't use the method SaveChangesAsync() as I can see method is decorated with the async keyword.

Reply:
changed.